git_machete/cmd.py

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
+from typing import Any, AnyStr, Callable, Dict, Generator, Iterable, Iterator, List, Match, Optional, Set, Tuple, TypeVar
```

Comment:
Whoops that's failing on pep8... but you probably already know

Reply:
Oh okay there some other erros so I'm not gonna go through that file in detail yet